doc/source/PreConfiguredSites.rst

```diff
@@ -388,6 +379,8 @@ As of spring 2023, there is an inconsistency in `libstdc++` versions on Acorn be
 NOAA Parallel Works (AWS, Azure, Gcloud)
 ----------------------------------------
 
+**Note. This section needs updating from EPIC**
+
```

Comment:
384-395 are fine.

i am not sure when/why the comment line 394 got in there, but i believe it can be removed so that the module load of ecflow just directly follows the mysql load.

Reply:
That's because I only installed ecflow in the AWS partition and then sent the entire contrib spack-stack stuff back to you. But it's great if you or Mark got it built on Azure and Gcloud as well in the meantime!

Reply:
I'll resolve this, and the other comment before - I added your test to the latest commit.